core/src/impl/Kokkos_Core.cpp

```diff
@@ -168,6 +168,9 @@ int get_ctest_gpu(const char* local_rank_str) {
 
 namespace {
 
+#if defined(KOKKOS_ENABLE_CUDA) || \
+    defined(KOKKOS_ENABLE_ROCM) || \
```

Comment:
Technically, this has no effect and the `ROCM` related implementations will be removed (in favor of `HIP`) in the future anyway. Since there are still a lot of places that have it, it doesn't matter that much, though.